src/main/java/bio/terra/service/dataset/flight/ingest/scratch/CreateScratchFileForGCPStep.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.service.dataset.flight.ingest.scratch;
+
+import static bio.terra.service.dataset.flight.ingest.IngestMapKeys.TABLE_NAME;
+
+import bio.terra.service.common.gcs.CommonFlightKeys;
+import bio.terra.service.common.gcs.GcsUriUtils;
+import bio.terra.service.job.DefaultUndoStep;
+import bio.terra.service.job.JobMapKeys;
+import bio.terra.service.resourcemanagement.google.GoogleBucketResource;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.StepResult;
+import com.google.cloud.storage.BlobId;
+
+public class CreateScratchFileForGCPStep extends DefaultUndoStep {
+
+  public CreateScratchFileForGCPStep() {}
+
+  @Override
+  public StepResult doStep(FlightContext context) throws InterruptedException {
+    FlightMap workingMap = context.getWorkingMap();
+    String tableName = context.getInputParameters().get(TABLE_NAME, String.class);
+    GoogleBucketResource bucket =
+        workingMap.get(CommonFlightKeys.SCRATCH_BUCKET_INFO, GoogleBucketResource.class);
+    BlobId scratchFilePath =
+        GcsUriUtils.getBlobForFlight(bucket.getName(), tableName, context.getFlightId());
```

Comment:
Are we not putting the file in an `ingest-scratch` directory in GCP like we are for Azure? I think it would make sense to share the same directory structure

Reply:
I'm following what we were already doing for scratch files.  I'm totally sure I follow.  Are you saying to add `/ingest-scratch/` in front of table name?

Reply:
Yeah. That seems to be what we're doing in the Azure instance of this step

Reply:
gotcha.  can do!